.travis.yml

```diff
@@ -1,3 +1,5 @@
+before_install:
+  - gem update bundler
```

Comment:
Stole this from https://github.com/seejohnrun/ice_cube/pull/321 - that build continued to fail because of an unrelated issue, addressed in this PR.